ui/page/components/account_selector.go

```diff
@@ -67,7 +65,7 @@ func (as *AccountSelector) Changed() bool {
 
 func (as *AccountSelector) Handle() {
 	for as.openSelectorDialog.Clicked() {
-		newAccountSelectorModal(as.Load, as.selectedAccount, as.wallets).
+		newAccountSelectorModal(as.Load, as.selectedAccount).
```

Comment:
Suggestion:
- Assign this modal to `as.selectorModal` when `openSelectorDialog` is clicked.
- When the modal is closed, set `as.selectorModal = nil`.
- When the proposed `as.listenForTxNotifications` sees a new tx or block, it calls `as. UpdateSelectedAccountBalance()` and `if as.selectorModal != nil`, then it also calls `as.selectorModal.setupAccountBalance()/setupAccounts` and refreshes the window.

One listener, does everything.